rails-commenting.rb

```diff
@@ -61,7 +61,7 @@ def destroy
   # 8) Private is a built in Rails statement that protects the classes beneath it
```

Comment:
not sure about this, I will past below the definition from the syllabus, I hope it will help! :) 

Private is a keyword in Ruby that restricts the scope of where a method can be called. Since strong params are only meant to be used as arguments to controller methods, we can list them as private to the controller class. That way there is no chance they can be called from somewhere else in the program.